lib/pure/times.nim

```diff
@@ -229,31 +232,29 @@ proc normalize[T: Duration|Time](seconds, nanoseconds: int64): T =
     result.seconds -= 1
   result.nanosecond = nanosecond.int
 
-proc initTime*(unix: int64, nanosecond: NanosecondRange): Time =
-  ## Create a ``Time`` from a unix timestamp and a nanosecond part.
-  result.seconds = unix
-  result.nanosecond = nanosecond
+# Forward declarations
+proc utcZoneInfoFromUtc(time: Time): ZonedTime {.tags: [], raises: [], benign .}
+  ## Forward declaration
+proc utcZoneInfoFromTz(adjTime: Time): ZonedTime {.tags: [], raises: [], benign .}
+  ## Forward declaration
+proc localZoneInfoFromUtc(time: Time): ZonedTime {.tags: [], raises: [], benign .}
+  ## Forward declaration
+proc localZoneInfoFromTz(adjTime: Time): ZonedTime {.tags: [], raises: [], benign .}
+  ## Forward declaration
+proc initTime*(unix: int64, nanosecond: NanosecondRange): Time 
+  {.tags: [], raises: [], benign noSideEffect.}
+  ## Forward declaration
+
+proc initDuration*(nanoseconds, microseconds, milliseconds,
+                   seconds, minutes, hours, days, weeks: int64 = 0): Duration 
```

Comment:
Maybe change the order of the parameters?
To ```proc initDuration*(weeks, days, hours, minutes, seconds, milliseconds, microseconds, nanoseconds: int64 = 0): Duration```
For compatibility with ```initDateTime```.

Reply:
The order is because of compatibility with `initTimeInterval`.  The proc was not added in this PR, only moved around.